### ISSUE TITLE
module 'tensorflow' has no attribute 'function' .

### ISSUE THREAD

**Issue by MrAbhishek319** [tensorflow/tensorflow#35604]
Name: tensorflow
Version: 2.0.0 in Anaconda with Python v. 3.7.3

**Comment by generationXcode** [comment]
Can you share the code?😊

**Comment by MrAbhishek319** [comment]
> Can you share the code?😊

# Notice the use of `tf.function`
# This annotation causes the function to be "compiled".
@tf.function
def train_step(images):
  seed = tf.random.normal([BATCH_SIZE, SEED_SIZE])

  with tf.GradientTape() as gen_tape, tf.GradientTape() as disc_tape:
    generated_images = generator(seed, training=True)

**Comment by amahendrakar** [comment]
@MrAbhishek319,
Could you please provide the link for the document you are referring to? Can you please elaborate the context as well? Thanks!

**Comment by amahendrakar** [comment]
Any updates regarding this issue? Thanks!

**Comment by MrAbhishek319** [comment]
I run the code through google colaberation and its working. I was running the code with Jupyter when I got the error.
